src/common/common.go

```diff
@@ -42,3 +44,20 @@ func SetConfiguringStatusForHosts(client inventory_client.InventoryClient, inven
 		}
 	}
 }
+
+func IsPodInStatus(k8Client k8s_client.K8SClient, podNamePrefix string, namespace string, labelMatch map[string]string, status v1.PodPhase, log logrus.FieldLogger) bool {
+	pods, err := k8Client.GetPods(namespace, labelMatch)
+	if err != nil {
+		log.WithError(err).Warnf("Failed to get pod with prefix %s in namespace %s", podNamePrefix, namespace)
```

Comment:
This will spam the log with errors no?

Reply:
possible but u don't want to see that u don't have a pod when u are trying to reach it?

Reply:
and for sure it must not happened